packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -124,16 +126,32 @@ MeetingInfoUtil.isPhoneNumber = (phoneNumber) => {
 };
 
 MeetingInfoUtil.getHydraId = (destination) => {
-  const decodedDestination = deconstructHydraId(destination).id;
+  const decodedDestination = deconstructHydraId(destination);
 
-  if (UUID_REG.test(decodedDestination)) {
-    return decodedDestination;
+  if (decodedDestination && UUID_REG.test(decodedDestination.id)) {
+    if (decodedDestination.type === _ROOM_) {
+      return {room: true, destination: decodedDestination.id};
```

Comment:
This functionality might be good to roll up into the `deconstructHydraId` function. Have it return another property of `room:true, person:false`. Not necessary but would be helpful. Even adding the UUID test with another property of `valid:true`.

Reply:
i'll take a look